apps/src/lib/ui/lti/link/LtiLinkAccountPage/cards/ExistingAccountCard/index.tsx

```diff
@@ -51,22 +52,26 @@ const ExistingAccountCard = () => {
         title={i18n.ltiLinkAccountExistingAccountCardHeaderLabel()}
         icon={
           <FontAwesomeV6Icon
-            className={classNames(styles.icon, 'fa-3x')}
+            className={classNames(styles.icon, 'fa-2x')}
             iconName={'user-check'}
           />
         }
       />
-      <CardContent className={cardStyles.cardContent}>
-        {i18n.ltiLinkAccountExistingAccountCardContent({
-          providerName: ltiProviderName,
-        })}
-      </CardContent>
+      <div className={classNames(styles.cardContentContainer)}>
+        <CardContent className={cardStyles.cardContent}>
+          {isLMS
+            ? i18n.ltiLinkAccountExistingAccountCardContent({
+                providerName: ltiProviderName,
+              })
+            : i18n.accountExistingAccountCardContentWorkshopEnroll()}
+        </CardContent>
+      </div>
```

Comment:
I haven't tested this, but I believe you can have the padding be added to the CardContent div

```suggestion
      <CardContent className={classNames(styles.cardContentContainer, cardStyles.cardContent)}>
        {isLMS
          ? i18n.ltiLinkAccountExistingAccountCardContent({
              providerName: ltiProviderName,
            })
          : i18n.accountExistingAccountCardContentWorkshopEnroll()}
      </CardContent>
```